### ISSUE TITLE
[a11y] Create hidden "back" button for separate page flows

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#4922]
This should be visually hidden from the user but accessible to screenreaders. Below the header of each separate page but above any errors, there should be a link that reads "Cancel" (or, more accurately, it should match whatever is read when the user tabs to "Cancel" at the bottom of separate page forms) to act as a skip link back to the main content.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1624)
┆Issue Number: FXA-1624

**Comment by data-sync-user** [comment]
➤ Lauren Zugai commented:

Marking this as canceled. 

Visual design has added a back button for the separate page views so we don't need a hidden button.
